src/main/java/com/bootcamp/demo/LoginController.java

```diff
@@ -1,7 +1,11 @@
 package com.bootcamp.demo;
 
+import com.bootcamp.model.Card;
 import org.springframework.stereotype.Controller;
 import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PostMapping;
```

Comment:
Please remove these unused imports. The best practice is to not alter classes that don't serve the scope of the current Pull Request thus keeping the change list clean and focused on the implemented business logic.